test/integration/windows/checkout/index.js

```diff
@@ -21,6 +21,10 @@ if (action === 'checkout') {
 
         return ZalgoPromise.try(() => {
 
+            if (window.xprops.onInit) {
```

Comment:
Would probably need to create a `windows/fields` -- this file is only used for the checkout component which has a different url and different props.